src/autogluon/bench/frameworks/multimodal/exec.py

```diff
@@ -193,6 +193,7 @@ def run(
         "problem_type": train_data.problem_type,
         "presets": params.pop("presets", None),
         "path": os.path.join(benchmark_dir, "models"),
+        "eval_metric": val_data.metric,
```

Comment:
`val_data` might not be available for all datasets so we need to handle that.

Reply:
How about add these lines:

```Python
if val_data is not None:
    predictor_args["eval_metric"] = val_data.metric
````

Reply:
Yes that would work.

Reply:
Done.